utility.sh

```diff
@@ -138,22 +138,14 @@ get_integration_resource() {
 }
 
 get_json_value() {
-  if [ "$1" == "" ] || [ "$2" == "" ]; then
```

Comment:
@shrivara because if you do not pass the `key name`, it will output the whole object.